docs/news.rst

```diff
@@ -2,6 +2,81 @@
 Release Notes
 =============
 
+2.3 (2021-02-XX)
+================
+
+Highlights
+----------
+
+- The `index_files_for_migration <synapseutils.html#synapseutils.migrate_functions.index_files_for_migration>`__ and
+  `migrate_indexed_files <synapseutils.html#synapseutils.migrate_functions.migrate_indexed_files>`__ functions are added
+  to synapseutils to help migrate files in Synapse projects and folders between AWS S3 buckets in the same region.
+  More details on using these utilities can be found `here <S3Storage.html#storage-location-migration>`__.
+
+- This version supports login programatically and from the command line using personal access tokens which can be obtained
```

Comment:
Change "which" to "that". 

"tokens which can be obtained" to " tokens that can be obtained"